types/index.d.ts

```diff
@@ -0,0 +1,137 @@
+export type Leagues = "little" | "great" | "ultra" | "master";
```

Comment:
There shouldn't be a `Leagues` type... In principle they can be arbitrary strings.